src/foam/nanos/auth/AgentAuthClient.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'AgentAuthClient',
+  extends: 'foam.nanos.auth.ProxyAuthService',
+  documentation: `
+  `,
```

Comment:
Can you provide documentation on what the client method decorator is doing

Reply:
It seems like the main purpose is to purge caches when the user calls auth.actAs(). You should mention that in the documentation.